# Hardcode python location to avoid rebuilding package when the python version change.

Index: git/awn-settings/awnSettings.py.in
===================================================================
--- git.orig/awn-settings/awnSettings.py.in	2011-07-14 11:39:40.000000000 +0200
+++ git/awn-settings/awnSettings.py.in	2011-07-14 11:45:16.000000000 +0200
@@ -27,8 +27,8 @@
 import pwd
 import getopt
 
-PLAT_PKG = '@PY_PLAT_PKG@'
-SITE_PKG = '@PY_SITE_PKG@'
+PLAT_PKG = '/usr/lib/python'+sys.version[0:3]+'/site-packages'
+SITE_PKG = '/usr/lib/python'+sys.version[0:3]+'/site-packages'
 AWN_SETTINGS_DIR = os.path.join('@PKGDATADIR@', 'awn-settings')
 sys.path = [AWN_SETTINGS_DIR] + sys.path
 if PLAT_PKG not in sys.path:
